cart.rb

```diff
@@ -0,0 +1,42 @@
+require_relative "store"
+require_relative "main"
+
+
+class Cart
+  attr_reader :order, :menu
```

Comment:
Remove attr_reader from menu. Adding an attribute reader to a variable creates public methods for it.